src/components/AeCard.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div :class="['ae-card', fill]">
+    <slot />
+    <ae-toolbar :fill="fill">
+      <slot name="toolbar" />
+    </ae-toolbar>
+  </div>
+</template>
+
+<script>
+import { AeToolbar } from '@aeternity/aepp-components-3';
+
+export default {
+  name: 'AeCard',
```

Comment:
Not necessary to set a component name, it is useful for:
- recursive components
- debugging if it is rendered anonymously (we are don't have stuff like this, usually, we registers components like  `components: { AeToolbar }` in this case the component name is `AeToolbar` even if `name` field is undefined in AeToolbar component)
- components package: for some reason documentation can't be built without `name` field

https://vuejs.org/v2/api/#name